tests/test_action.py

```diff
@@ -197,3 +197,23 @@ def test_year_filter_value(global_var):
         "T00:00:00.000000000" not in vis.to_Altair()
     ), "Year filter title contains extraneous string, not displayed as summarized string"
     df.clear_intent()
+
+
+def test_similarity(global_var):
+    df = pytest.car_df
+    df["Year"] = pd.to_datetime(df["Year"], format="%Y")
+    df.set_intent(
+        [
+            lux.Clause("Year", channel="x"),
+            lux.Clause("Displacement", channel="y"),
+            lux.Clause("Origin=USA"),
+        ]
+    )
+    df._repr_html_()
+    assert len(df.recommendation["Similarity"]) == 3
+    assert (
+        df.recommendation["Similarity"][0].score
+        > df.recommendation["Similarity"][1].score
```

Comment:
This is not really testing anything regarding the similarity metric, since if we are sorting correctly, the first score should always be bigger than the second, than the third. What we should have is having the visualization that we know "look" similar and compare the score of those instead.